conda.recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: nb_conda_kernels
-  version: "0.1.1"
+  version: "0.1.2"
```

Comment:
can we get this out of here, and use an environment variable? Its just _one more place_ to remember to update at release time: example from [blaze](https://github.com/blaze/blaze/blob/master/all.recipe/meta.yaml)...

```yaml
  version: {{ environ.get('GIT_DESCRIBE_TAG', '')}}
```